tests/unit/utils/test_thin.py

```diff
@@ -472,9 +468,9 @@ def test_gen_thin_python_exist_or_not(self):
                               'tops for Python 2',
                               handler.messages)
 
-    @skipIf(thin._six.PY2, reason='This test does too much patching on places it should not. Needs to be reworked')
+    @pytest.mark.skip('This test does too much patching on places it should not. Needs to be reworked')
```

Comment:
I really dont want this test to get lost. Can we add a similar message to the WARROOM and specify it needs to be re-worked? Or are we using this similar message for all tests that are over mocked?

Reply:
We can append the WAR ROOM

Reply:
Done